### PR TITLE
No data - home page

### PR DESCRIPTION
**Pull request by jjaykim**
`for문`,,,,,,😢

### CODE REVIEW

File: client/src/components/home_content/HomeContent.tsx [hunk 1]
```diff
@@ -16,13 +17,29 @@ import { GallerySlide } from '../gallery_slide/GallerySlide';
...
+      pos: i + 1,
+      name: '-',
+      played: '-',
+      GD: '-',
+      points: '-',
+    });
+  }
+  return standingDefaultData;
+};
+
```
Comment (koreancanadiansoccer) [inline client/src/components/home_content/HomeContent.tsx:42]: no need to loop - this is gonna run everytime it re-render I think.
Just hard code it and put it in /defaultData.ts as well

```
export const standingDefaultData = [
{pos:1, name: '-', playerd: '-', GD: '-', points:'-' }
{pos:2, name: '-', playerd: '-', GD: '-' ,points:'-' }
]
```

File: client/src/components/home_content/HomeContent.tsx [hunk 2]
```diff
@@ -50,6 +67,20 @@ const generateStandingData = (leagueTeams: LeagueTeam[]) => {
...
+  for (let i = 0; i < 4; ++i) {
+    scorerDefaultData.push({
+      pos: i + 1,
+      name: '-',
+      club: '-',
+      goals: '-',
+    });
+  }
+  return scorerDefaultData;
```
Comment (koreancanadiansoccer) [inline client/src/components/home_content/HomeContent.tsx:81]: same here

File: client/src/components/home_content/HomeContent.tsx (hunk 1 above)
Comment (jjaykim) [reply client/src/components/home_content/HomeContent.tsx:42]: 안그래도 고치고 있었습니다 ! ㅋㅋㅋㅋㅋ 다시 올리겠습니다 !

File: client/src/components/home_content/HomeContent.tsx [hunk 3]
```diff
@@ -129,48 +164,91 @@ const UnstyledHomeContent: FunctionComponent<HomeContentProps> = ({
...
-              {map(viewer.leagueAgeKeys, (leagueAge) => (
-                <Box key={`home-table-league-selection-${leagueAge}`}>
-                  <LeagueSelect
-                    title={leagueAge}
-                    selected={tableAgeType === leagueAge}
-                    onClick={() => setTableAgeType(leagueAge)}
-                  />
-                </Box>
-              ))}
+              {isEmpty(viewer.leagueAgeKeys)
```
Comment (koreancanadiansoccer) [inline client/src/components/home_content/HomeContent.tsx:167]: why yoou checking this everywhere. 
Just create new memo var and set it there.
const leagueAgeKeyOptions = useMemo(... check if it's empty ? default: viewer.leagueagekeys )

Comment (jjaykim) [reply client/src/components/home_content/HomeContent.tsx:167]: 네네! 며칠 안 했다고,,, 머리가 굳었나봅니다,,,,,다 바꾸고 있습니다 지금ㅋㅋㅋㅋ

**Pull request merged**